src/main/java/core/basesyntax/file/FileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.file;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String filePath) {
```

Comment:
Remember to follow the encapsulation principle. The method 'write' should not be public if it's not intended to be exposed as part of the API.